compiler/src/dmd/tokens.h

```diff
@@ -133,6 +133,7 @@ enum class TOK : unsigned char
     // Leaf operators
     identifier,
     string_,
+    interpolated,
```

Comment:
https://github.com/dlang/dmd/pull/15722#discussion_r1367852557

Reply:
[ibuclaw](https://github.com/ibuclaw) [36 minutes ago](https://github.com/dlang/dmd/pull/15722#discussion_r1367890971)
Should be : compiler/src/dmd/token.h, not cxxfrontend.cc.


but that's what i did....

Reply:
That's for compatibility with C++ users of DMD. You also need to update the stuff mentioned in the comments to ensure that the DMD C++ 'unit' test works. You will note that currently the tests fail..

Reply:
There are no comments in that file, nor in the .d file. The comment on github says "corresponding header"... what is that.

And yes, I see the tests fail, if I knew how to fix it, I would have done it already.

Reply:
ok i saw expression.h let's see how that goes.

kinda remarkable how much duplication there is in dmd. two asts, two languages, golly

Reply:
> corresponding header

`foo.d` corresponds to `foo.h`